rpc/README.md

```diff
@@ -0,0 +1,96 @@
+# RPC
+
+**RPC** is an externally-facing component through which clients can interact with the node. It receives client requests 
+(e.g., to synchronize with the latest state of the chain, or to submit transactions), performs basic validation, 
+and forwards the requests to the appropriate internal components.
+**RPC** is one of components of the [Miden node](..).
+
+## Architecture
+
+The Miden node is still under heavy development and current architecture is subject of change. This topic will be 
+filled later.
+
+## Usage
+
+### Installing RPC
+
+RPC can be installed and run as a part of [Miden node](../README.md#installing-the-node).
```

Comment:
Change to: The RPC can be installed and run as part of [Miden node](https://github.com/0xPolygonMiden/miden-node/pull/README.md#installing-the-node).
But if you intend on running the RPC as a separate process, you will need to install and run it as follows: